src/main/java/de/egga/Thing.java

```diff
@@ -10,10 +10,9 @@
 
     public Integer getValue() {
         return value;
-	value = 10; 
    }
 
     public void setValue(final Integer value) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "value". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://gkuzmishch-ltm2.internal.salesforce.com:9000/coding_rules#rule_key=squid%3AS1172)